controllers/admin/AdminImagesController.php

```diff
@@ -178,6 +192,51 @@ public function __construct()
                 ],
                 'submit' => ['title' => $this->trans('Save', [], 'Admin.Actions')],
             ],
+            'optimization' => [
+                'title' => $this->trans('Advanced optimization options', [], 'Admin.Design.Feature'),
+                'icon' => 'icon-picture',
+                'top' => '',
+                'bottom' => '',
+                'description' => $this->trans('Optimize bandwidth and disk usage.', [], 'Admin.Design.Help'),
```

Comment:
@Julievrz That settings are for admins, you will go there once in 5 years, I suggest don't invest much time. :-)

Reply:
In my opinion, this is the best design: simple boolean options.

Reply:
Yes I am not questioning the toggle buttons but this: 
![image](https://user-images.githubusercontent.com/70583503/150152835-e9a3664b-5f76-4530-8f61-3c6ceb58bc8d.png)
I think it could be hidden in a tooltip next to the section's name, as it was recently done on the translation page. 

Well, this is only a suggestion. 😃